views/summary/view.blade.php

```diff
@@ -26,6 +43,7 @@
 							<p class="w-100 text-wrap"><em>{!! $order->comment !}}</em></p>
```

Comment:
{!! $order->comment !!}}

Missing !

Reply:
Did not even notice that! And should have been caught when merged. Now fixed and a new PR created. Thank you!